salt/journal-cms/api-dummy.sls

```diff
@@ -0,0 +1,9 @@
+api-dummy-nginx-vhost-dev:
```

Comment:
also needs to replicate in `pillar/journal-cms.sls` this couple of lines https://github.com/elifesciences/search-formula/blob/master/salt/pillar/search.sls#L12 which will tell the `api-dummy-formula` we are depending upon that it's not the main project